src/main/java/bhs/devilbotz/utils/ShuffleboardManager.java

```diff
@@ -109,7 +108,7 @@ private void initAutoModePreferences() {
   /** Updates the values on the shuffleboard. */
   public void updateValues() {
     // update gripper setpoint using network tables
-    gripperSetpoint.setBoolean(Gripper.getAtSetpoint());
+    // gripperSetpoint.setBoolean(Gripper.getAtSetpoint());
```

Comment:
What's the change here?

Reply:
Because I commented out lines earlier 51-57, had to comment this out too.  I had no good idea/solution to make this work at the time.  It's only a dashboard display, not critical to robot functioning.  Figured we could try to solve this later, or just remove it if not needed.